README.md

```diff
@@ -1,43 +1,110 @@
+
+All Bitrise CLI tools are written in Go.
 
-**We now started to move our Sample repositories to a dedicated GitHub account: https://github.com/bitrise-samples**
+## Stack
 
-Feel free to fork them and test them on [Bitrise.io](https://www.bitrise.io).
+### OS X
 
+We are working with [vSphere](https://www.vmware.com/products/vsphere), [Ansible](http://docs.ansible.com/) and [vagrant](https://www.vagrantup.com/docs/) to virtualize, bootstrap and fire up the OS X build machines for builds on [bitrise.io](https://www.bitrise.io).
 
-## Contribution to bitrise stack tools & steps
+You can find all of our bootstrap scripts on the [OS X bootstrap GitHub page](https://github.com/bitrise-io/osx-box-bootstrap).
 
-If you want to collaborate with us creating useful automation tools and steps please go to the https://github.com/bitrise-io/bitrise-contrib repository and follow the guide.
+### Docker
 
-You can help us work on existing and new Steps or tools, and you can keep track of others' projects through the `contrib` repository (https://github.com/bitrise-io/bitrise-contrib).
+We have a [base image](https://github.com/bitrise-docker/bitrise-base) and an [Android version](https://github.com/bitrise-docker/android) with pre-installed tools for Android builds.
 
+>When a new version of a stack is available there will be a system report generated under the [system_reports](https://github.com/bitrise-io/bitrise.io/blob/master/system_reports) folder of this repository.
+
+## Contribution to Bitrise stack, tools & steps
+
+If you want to collaborate with us creating useful automation tools and steps please go to the [`contrib`](https://github.com/bitrise-io/bitrise-contrib) repository and follow the guides. you can keep track of others' projects there as well.
+
+### Request a tool to be pre-installed on a build machine
+
+* For OS X you can create an issue or pull request at [`osx-box-bootstrap`](https://github.com/bitrise-io/osx-box-bootstrap)
+* For our Docker based stack you can create an issue or pull request at:
+    * [`bitrise-base`](https://github.com/bitrise-docker/bitrise-base), for generic tools and configurations (which are not related to Android)
+    * [`android`](https://github.com/bitrise-docker/android), for Android related tools and configurations
+
+## Sample Apps
+
+Check out our sample app repositories.
+
+**iOS**
+
+- [Xcode 7 sample project (with UI Testing)](https://github.com/bitrise-io/sample-apps-ios-xcode7)
+- [Quick sample project](https://github.com/bitrise-io/sample-test-ios-quick)
+- [XCTest sample project](https://github.com/bitrise-io/sample-test-ios-xctest)
+- [Specta sample project](https://github.com/bitrise-io/sample-test-ios-specta)
+- [Kiwi sample project](https://github.com/bitrise-io/sample-test-ios-kiwi)
+
+**Android**
+
+- [with code sign](https://github.com/bitrise-samples/android-sdk22-code-sign)
+- [without gradlew](https://github.com/bitrise-samples/android-sdk22-no-gradlew)
```

Comment:
this is just a test repository, use the standard "sample-apps-android-sdk22" instead ;)